ide/provisioning/install-debian-packages.sh

```diff
@@ -13,9 +13,10 @@ echo Installing base dependencies...
 apt-get install $APTOPTS make \
   librsvg2-bin xsltproc \
   imagemagick gettext ffmpeg \
-  git quilt zip \
+  git quilt unzip zip \
   m4 automake wget \
-  ttf-bitstream-vera fakeroot
+  ttf-bitstream-vera fakeroot \
+  ccache
```

Comment:
What for?

Reply:
Removed, a left-over I didn't tidy up (introduced without explanation in https://github.com/lordfolken/xcsoar-docker-build/pull/1).